### ISSUE TITLE
Better accessibility for the profile fields

### ISSUE THREAD

**Issue by MarcoZehe** [nolanlawson/pinafore#503]
Right now, the different fields and their values for each field are just divs without any further semantics. Some screen readers lump these together into an unintelligible string.

Suggestion:

1. Give each div with selector "account-profile-meta-name" a role of "term" and a unique ID.
2. Give the corresponding div with selector "account-profile-meta-value" a role of "definition" and an aria-labelledby with the idref of the "term" from step 1.

An alternative would be to use "table" for the container "account-profile-meta" and "gridcell" for each of the "account-profile-meta-cell" divs, but we are missing the rows here that would be required to use with valid WAI-ARIA parent/child/grandchild relations, so I am suggesting the above instead to get this read better and give it more semantic meaning.

**Comment by nolanlawson** [comment]
Thanks, this was a really dumb oversight on my part! Unfortunately it would be tricky to use a `<table>` because I'm using CSS grid for this (probably overkill... as I'm not using anything other than a two-column layout).

I like the `role=term` and `role=description` approach as it seems to exactly describe what I want, while leaving the layout flexible in the future if I do decide to mix up the grid a bit. I found this definition in the WAI ARIA authoring practices, I'll give it a spin! https://www.w3.org/TR/wai-aria-1.1/#term

**Comment by nolanlawson** [comment]
temp demo version here: https://pinafore-wrntcozavh.now.sh

**Comment by nolanlawson** [comment]
fixed in https://github.com/nolanlawson/pinafore/pull/505
